src/data/customer/__test__/job.test.js

```diff
@@ -45,3 +46,119 @@ describe('Show job', async () => {
     }
   });
 });
+
+describe('job create', async () => {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+  let token;
+  let routePayload;
+  let route1;
+  let route2;
+  let routeIds;
+
+  beforeAll(async () => {
+    token = await getTokenAsync(
+      CONFIG.email,
+      CONFIG.password,
+      CONFIG.clientId,
+      CONFIG.clientSecret
+    );
+
+    routePayload = {
+      driverId: 10714,
+      pickupDate: CONFIG.jobTest.date,
+      route_settings: '{}',
+    };
+
+    // create route 1 now
+    const route1Res = await storeRouteAsync(
+      {
+        routes: [
+          {
+            ...routePayload,
+            routeLocations: [
+              {
+                sequence: 1,
+                orderId: CONFIG.jobTest.orderIds[0],
+                locationTypeId: 3,
+              },
+              {
+                sequence: 2,
+                orderId: CONFIG.jobTest.orderIds[0],
+                locationTypeId: 2,
+              },
+            ],
+          },
+        ],
+        replaceAllExisting: false,
+      },
+      token.accessToken
+    );
+    route1 = route1Res.data[0];
+
+    //
+    const route2Res = await storeRouteAsync(
+      {
+        routes: [
+          {
+            ...routePayload,
+            driverId: 10715,
+            routeLocations: [
+              {
+                sequence: 1,
+                orderId: CONFIG.jobTest.orderIds[1],
+                locationTypeId: 3,
+              },
+              {
+                sequence: 2,
+                orderId: CONFIG.jobTest.orderIds[1],
+                locationTypeId: 2,
+              },
+            ],
+          },
+        ],
+        replaceAllExisting: false,
+      },
+      token.accessToken
+    );
+    route2 = route2Res.data[0];
+
+    routeIds = `${route1.id},${route2.id}`;
+  });
+
+  it('Should create a job', async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
+    let response;
+    try {
+      response = await createJobsAsync(routeIds, token.accessToken);
+    } finally {
+      expect(response).toHaveLength(2);
+      expect('job' in response[0]).toBeTruthy();
+      expect('job' in response[1]).toBeTruthy();
+      const job1 = response[0].job;
+      const job2 = response[1].job;
+      expect('id' in job1).toBeTruthy();
+      expect('id' in job2).toBeTruthy();
+    }
+  });
+  it('should be failure', async () => {
```

Comment:
what should be failure?

Reply:
@maartenhemmes  added more description for the failure case